Makefile

```diff
@@ -13,7 +13,10 @@ install-deps:
 	apt-get install --no-install-recommends -y build-essential clang protobuf-compiler ocl-icd-opencl-dev aria2 cmake
 
 install-lint-tools:
```

Comment:
It will be great to also change `Install MDBook` step in `docs.yml`

Reply:
Good idea!